tests/helpers.py

```diff
@@ -1572,3 +1572,32 @@ def wait_for_ct_pod_recovery():
         else:
             return False
     return True
+
+
+def label_worker_node(node_list, label_key, label_value):
+    """
+    Function to label worker node for running app pods on specific worker nodes.
+
+    Args:
+        node_list (list): List of node name
+        label_key (str): Label_key to be added in worker
+        label_value (str): Label_value
+    """
+    ocp_obj = OCP()
+    for node in node_list:
+        out = ocp_obj.exec_oc_cmd(command=f"label node {node} {label_key}={label_value}", out_yaml_format=False)
+        logger.info(out)
+
+
+def remove_label_from_worker_node(node_list, label_key):
+    """
+    Function to remove label from worker node.
+
+    Args:
+        node_list (list): List of node name
+        label_key (str): Label_key to be remove from worker node
+    """
+    ocp_obj = OCP()
+    for node in node_list:
```

Comment:
haven't tried but probably will also work in one command

Reply:
@petr-balogh it works, let me update the patch.

------------------------------------------------------------------------------- live log call -------------------------------------------------------------------------------
17:28:54 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/2ndJan/auth/kubeconfig get Node  --selector=node-role.kubernetes.io/worker -o yaml
17:28:56 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/2ndJan/auth/kubeconfig label node ip-10-0-141-16.us-east-2.compute.internal ip-10-0-158-150.us-east-2.compute.internal ip-10-0-174-181.us-east-2.compute.internal nodetype=app-scale
17:29:01 - MainThread - tests.helpers - INFO - node/ip-10-0-141-16.us-east-2.compute.internal labeled
                                               node/ip-10-0-158-150.us-east-2.compute.internal labeled
                                               node/ip-10-0-174-181.us-east-2.compute.internal labeled
17:29:01 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/2ndJan/auth/kubeconfig label node ip-10-0-141-16.us-east-2.compute.internal ip-10-0-158-150.us-east-2.compute.internal ip-10-0-174-181.us-east-2.compute.internal nodetype-
17:29:05 - MainThread - tests.helpers - INFO - node/ip-10-0-141-16.us-east-2.compute.internal labeled
                                               node/ip-10-0-158-150.us-east-2.compute.internal labeled
                                               node/ip-10-0-174-181.us-east-2.compute.internal labeled
PASSED                                                                                                                                                                [100%]

============================================================================ 1 passed in 26.09s =============================================================================